### PR TITLE
fix(translation): errors in StudyList and StudyBrowser for es

### PR DESCRIPTION
**Pull request by juaesto**
### Changes & Results
There are some translations errors in StudyList and StudyBrowser, I have detected when set the viewer in Spanish translations. There are some errors in spanish translations. #3169 

#### PR

- [X] My Pull Request title is descriptive, accurate and follows the
  semantic-release format and guidelines.

#### Code

- [X] My code has been well-documented (function documentation, inline comments,
  etc.)

#### Public Documentation Updates

- [X] The documentation page has been updated as necessary for any public API
  additions or removals.

#### Tested Environment

- [X] "OS: Windows 10
- [X] "Node version: 16.14.2
- [X] "Browser: Chrome 109.0.5414.120
  

[blog]: https://circleci.com/blog/triggering-trusted-ci-jobs-on-untrusted-forks/
[script]: https://github.com/jklukas/git-push-fork-to-upstream-branch

### CODE REVIEW

File: platform/i18n/src/locales/en-US/index.js [hunk 1]
```diff
@@ -1,25 +1,23 @@
-import AboutModal from './AboutModal.json';
 import Buttons from './Buttons.json';
 import CineDialog from './CineDialog.json';
 import Common from './Common.json';
-import DatePicker from './DatePicker.json';
 import Header from './Header.json';
 import MeasurementTable from './MeasurementTable.json';
+import SidePanel from './SidePanel.json';
+import StudyBrowser from './StudyBrowser.json';
```
Comment (sedghi) [inline platform/i18n/src/locales/en-US/index.js:7]: why we are removing AboutModal and DatePicker etc. from this file?

Comment (juaesto) [reply platform/i18n/src/locales/en-US/index.js:9]: Excuse Me, it's true, this is an error. I have pushed the fix 889a71f

Comment (sedghi) [reply platform/i18n/src/locales/en-US/index.js:9]: Thanks, also the viewportDownloadForm

Comment (juaesto) [reply platform/i18n/src/locales/en-US/index.js:9]: Done :)

**Pull request merged**